src/shared/utils/exportData/getFlankerRecords.ts

```diff
@@ -121,7 +121,7 @@ const getResponseObj = ({
     responseTouchTimestamp =
       'response_touch_timestamp' in response
         ? response.response_touch_timestamp || DEFAULT_VALUE
-        : videoDisplayRequestTimestamp + response.duration;
+        : videoDisplayRequestTimestamp + (<DecryptedFlankerAnswerItemValue>response).duration;
```

Comment:
it has type declaration in row 85, why it does not work?

Reply:
I get such an error "Property 'duration' does not exist on type 'never'"